DOCKER.md

```diff
@@ -6,7 +6,7 @@
 
 ```
 docker-compose up
-open http://localhost:3001
+open http://localhost:3000
```

Comment:
I shouldn't have changed this — should leave the original Klaxon docs untouched.